packages/extensions/venia-sample-language-packs/i18n/fr_FR.json

```diff
@@ -0,0 +1,366 @@
+{
+    "About Us": "À propos de nous",
+    "Account": "Compte",
+    "accountChip.chipText": "Salut, {name}",
+    "accountInformationPage.accountInformation": "Information sur le compte",
```

Comment:
Because there are multiple informations on the Account page, we always put a "**s**" to "**Informations**" in french.
In the french translations of Magento 2, there is also a "**s**".